build.sbt

```diff
@@ -8,11 +8,10 @@ scalaVersion := "2.13.10"
 libraryDependencies ++= Seq(
   guice,
   filters,
+  "ch.qos.logback" % "logback-classic" % "1.3.5",
```

Comment:
A transitive dependency upgrades slf4j to 2.x, but Play still ships with 1.7 and logback-classic 1.2, which is not compatible with slf4j 2.x yet. Upgraind logback-classic to 1.3 make that work